### DISCUSSION TITLE
Hide next button

### DISCUSSION THREAD

**Discussion by davidshumway** [gilbarbara/react-joyride#1082] [General]
There are props for hiding the skip button and back button, but is not a prop to hide the next button. As an example scenario, assume that you want the user to click on an item, such as a menu, in order to continue the tour. Assume you want to force the user to do so, that is, the user cannot skip by using the next button. But you also want to still show the back button.

A workaround is to use styles and hide the next button:

```javascript
styles: {
    buttonNext: { // no prop to hide next!
        display: 'none'
    }
}
```

**Comment by gilbarbara** [comment]
Hey @davidshumway 

If you need additional customization, you can use a custom tooltip component.
